subsys/logging/log_output.c

```diff
@@ -38,7 +39,7 @@ static const char *const colors[] = {
 	NULL,
 	LOG_COLOR_CODE_RED,     /* err */
 	LOG_COLOR_CODE_YELLOW,  /* warn */
-	NULL,                   /* info */
+	LOG_COLOR_CODE_GREEN,   /* info */
```

Comment:
looks good but people got used to how it is (also adding color gives additional overhead). Please add it as an option to `Kconfig.formatting`. Then here just do:

`IS_ENABLED(CONFIG_LOG_INFO_COLOR_GREEN) ? LOG_COLOR_CODE_GREEN : NULL`

Reply:
Done.

I don't understand why people got used to info color being same color as debug. Is this some legacy zephyr-specific stuff?
Almost all CLI tools use green color for information, I thought it's widely accepted by now.

Reply:
By default debug logs are disabled so info level can already be distinguished from other log levels. Additionally, coloring brings overhead of few bytes.